commands/bootstrap/resources/topology/topology_infrastructure_openstack.tmpl

```diff
@@ -1,6 +1,7 @@
 {{ define "Infrastructure" }}
-      type: org.ystia.yorc.infrastructure.OpenStackConfig
-      properties: {{formatAsYAML .Infrastructures.openstack 8}}
+      type: org.ystia.yorc.location.OpenStackConfig
```

Comment:
Why type is `org.ystia.yorc.location.OpenStackConfig` here while `org.ystia.yorc.infrastructure.GoogleConfig` config for gcp and others?